Targets/Sample/Sources/03MockUp/Waiter.swift

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
From here, the task is consist of `Mockup test code`.
Please collect `one topic` if we possible when open PR. 
Because simple PR has pros ..
- More readable
- Easy to roll-back
- The review scope is narrowed, making it easier to reflect feedback